### ISSUE TITLE
floating_panes does not inherit cwd from tab

### ISSUE THREAD

**Issue by scabala** [zellij-org/zellij#3364]
# 2. Issues with the Zellij UI / behavior / crash

## Issue description

When defining `floating_panes` inside `tab`, `pane`s in `floating_panes` do not inherit `cwd` from `tab`, as others parts of layout do.

## Minimal reproduction

Having following layout:
```kdl
layout cwd="/tmp" {
    tab name="Tab #1" cwd="/mnt" {
        pane name="Pane #1" {

        }
        floating_panes {
            pane name="Floating pane #1"
        }
    }

}
```
run it via `zellij -l ./layout.kdl`, nomal pane (`Pane #1`) will have working directory set to `/mnt`. However, `Floating pane #1` will have current working directory set to `/tmp`.

I think user expectation would be to have current working directory set for floating panes to `/mnt`.

**Comment by rockboynton** [comment]
just noticed this as well. And it doesn't seem to just be within a layout. Even if I run the command line action it ignores the cwd option 
`zellij action new-pane --cwd=/home/rboynton/sources --floating` just starts at `/home/rboynton`

**Comment by mikebcbc** [comment]
Same here - command line action and floating both not working.

**Comment by scabala** [comment]
Fixed in 0.41.2 so closing
